Junaid_Paracha/Assignments/As_libary.js

```diff
@@ -0,0 +1,119 @@
+//Can we make this into a method of an object?
```

Comment:
Yes! you could have placed this into underscore. var _ = { each : function(arr, callback) {..}, map: etc};